picard/acoustid/__init__.py

```diff
@@ -113,12 +113,22 @@ def _on_lookup_finished(self, task, document, http, error):
                                 parsed_recording['score'] = score * result_score
                                 parsed_recording['acoustid'] = result['id']
                                 recording_list.append(parsed_recording)
-                        log.debug("AcoustID: Lookup successful for '%s'", task.file.filename)
 
-                    # Set AcoustID in tags if there was no matching recording
-                    if results and not recording_list:
-                        task.file.metadata['acoustid_id'] = results[0]['id']
-                        task.file.update()
+                    if results:
+                        if not recording_list:
+                            # Set AcoustID in tags if there was no matching recording
+                            task.file.metadata['acoustid_id'] = results[0]['id']
+                            task.file.update()
+                            log.debug(
```

Comment:
Actually the message was not correct, sorry. It's not about the threshold here. This just catches the case where there is an AcoustID that is not matched to any MB recording (as it can happen if AcoustID submission is done with a tool other than Picard, or when all links to recordings get removed)

Reply:
Ah yes, fixed in c6b7067